src/fragments/lib-v1/geo/js/geofences.mdx

```diff
@@ -6,7 +6,7 @@
 
 ## Provisioning geofence resources
 
-First, make sure you've provisioned a geofence collection resource and configured your app using the instructions in either [Configure a Geofence Collection]() or [Use existing Amazon Location Service resources](../existing-resources) and you have already setup [displaying a map](../maps) in your application.
+First, make sure you've provisioned a geofence collection resource and configured your app using the instructions in either [Configure a Geofence Collection](/[platform]/build-a-backend/more-features/geo/configure-geofencing) or [Use existing Amazon Location Service resources](../existing-resources) and you have already setup [displaying a map](../maps) in your application.
```

Comment:
since this is in lib-v1, should it be linking to prev?

Reply:
@heatheramz - will there be content at `/[platform]/prev/build-a-backend/more-features/geo/configure-geofencing`?